### PR TITLE
Feature/scrum page

### PR DESCRIPTION
**Pull request by JoNB7777**

### CODE REVIEW

File: app/views/scrum.scala.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+@main("Scrum description") {
+    <div id = "scrumDescription">
+        <h1 style="text-align: center;"> Scrum </h1>
+        <h3> What is Scrum? </h3>
+        <p> The Scrum framework consists of Scrum Teams and their associated roles, events, artifacts, and rules.
+            Each component within the framework serves a specific purpose and is essential to Scrum’s success and usage. <br>
+            Scrum employs an iterative, incremental approach to optimize predictability and control risk.
+            Three pillars uphold every implementation of empirical process control: transparency, inspection, and adaptation.
+        </p>
+        <p> An entity that adapts the Scrum framework will consist of self-organising Scrum teams which typically consist of 3 to 9 people.
```
Comment (DannyPennington) [inline app/views/scrum.scala.html]:
```suggestion
        <p> An entity that adopts the Scrum framework will consist of self-organising Scrum teams which are typically comprised of 3 to 9 people.
```

File: app/views/scrum.scala.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            When implementing a feature is assigned to a member of a Scrum team, this member is the only responsible for deciding how to implement.
```
Comment (DannyPennington) [inline app/views/scrum.scala.html]:
```suggestion
            When implementing a feature, it is assigned to a member of the Scrum team. This member is the only person responsible for deciding how to implement it.
```

File: app/views/scrum.scala.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+            However, they can of course seek advice from others on how to do this.
+        </p>
+        <p> Using Scrum allows for higher flexibility and thus adapting to problems more quickly.
```
Comment (DannyPennington) [inline app/views/scrum.scala.html]:
```suggestion
        <p> Using Scrum allows for higher flexibility and thus makes rapidly adapting to problems more achievable.
```

File: app/views/scrum.scala.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        <h3> The Scrum elements we used: </h3>
+        <ul>
+            <li> 3-day sprints </li>
+            <li> Sprint planning meetings </li>
+            <li> Daily standups </li>
+            <li> Sprint review meetings </li>
+            <li> Trello Board, including Sprint and Product Backlogs </li>
+        </ul>
+        <h3> How did using Scrum affect our project?  </h3>
+        <p> Using Scrum allowed us to localize issues in our code quickly, as we were allows only adding small
```
Comment (DannyPennington) [inline app/views/scrum.scala.html]:
```suggestion
        <p> Using Scrum allowed us to localize issues in our code quickly, as we were only adding small
```

File: app/views/scrum.scala.html [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+            increments to our working product, and could thus quickly determine which increment caused issues. <br>
+            The high amount of coordination and feedback, that is encouraged by using Scrum elements, such as Daily Standups,
```
Comment (DannyPennington) [inline app/views/scrum.scala.html]:
```suggestion
            The high amount of coordination and feedback that is encouraged by using Scrum elements, such as Daily Standups,
```

**Review by DannyPennington — CHANGES_REQUESTED** [review]
Just a couple of minor text changes from me, looks good overall though

**Review by AIbrahimQA — APPROVED** [review]
Looks fine after those changes

**Review by DannyPennington — APPROVED** [review]

**Pull request merged**